sklearn/inspection/tests/test_partial_dependence.py

```diff
@@ -532,6 +557,7 @@ def test_hist_gbdt_sw_not_supported():
 
 # TODO: Remove in 0.24 when DummyClassifier's `strategy` default updates
 @ignore_warnings(category=FutureWarning)
```

Comment:
here we should be more specifc as you do on the next line.

Reply:
Done in https://github.com/scikit-learn/scikit-learn/pull/16619/commits/5c3b713561a34ceb3afbff2d2553bec3e849f26d